### PR TITLE
add Leveret Pipeline

### PR DESCRIPTION
**Pull request by claysmalley**
Adds the Leveret Pipeline, an NGL transmission line in Enterprise Products' Hobbs East Gathering System.

### CODE REVIEW

**Comment by bairdlangenbrunner** [comment]
Hi Clay, this is looking pretty good but I did find a couple malformed linestrings, and deleted them in a second merge, [`948e4e7`](https://github.com/GlobalEnergyMonitor/GOIT-GGIT-pipeline-routes/pull/12/commits/948e4e76f0a96418ee873b507e2ce94d0772f957).

Could you take a look at those deletions and let me know you're cool with them? If so, I can merge this. And if you'd like me to do this a different way in the future, just let me know!

File: data/individual-routes/liquid-pipelines/P6505.geojson [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                "name": "Leveret Pipeline"
+            },
+            "geometry": {
+                "type": "LineString",
+                "coordinates": [
+                    [
+                        -103.90634180188,
+                        31.80982207214
+                    ],
+                    [
```
Comment (claysmalley) [inline data/individual-routes/liquid-pipelines/P6505.geojson:20]:
```suggestion
                "coordinates": [
                    [
```

You can make inline suggestions when reviewing diffs. Click and drag to select lines of code, and hit Ctrl+G. Here's an example (please don't merge) that deletes the first coordinate from this LineString.

**Review by claysmalley — COMMENTED** [review]
Ugh! Good catch. Go ahead and merge.

File: data/individual-routes/liquid-pipelines/P6505.geojson (hunk 1 above)
Comment (bairdlangenbrunner) [reply data/individual-routes/liquid-pipelines/P6505.geojson:20]: Got it, thanks for pointing that out!

**Pull request merged**